src/components/UserFollowingList.tsx

```diff
@@ -25,7 +25,7 @@ const UserFollowingList = ({ following, onUnfollow } : UserFollowingListProps) :
         <Flex data-testid="following-list" direction="column">
             { following.map(follow => (
                 <Flex
-                    key="follow.profile.id"
+                    key="follow.organization.id"
```

Comment:
The mistake here wasn't the content of the string, but the fact that it's a constant string literal, i.e. the same exact string key will be used for every organization, not the ID. I'm actually surprised this even renders more than one organization, and doesn't throw warnings in the console (or maybe it does?).

Reply:
Okay! I'm not sure I follow though. `follow.profile.id ` is different on each object, three different organizations are rendered and I don't get any warnings in the console.

Reply:
I verified that I do indeed get a warning in the console. See below. You should be getting the same warning, in the browser when the component is rendered there, or in the server log when the component is rendered on the server.

![image](https://user-images.githubusercontent.com/550212/118276956-8b645780-b4c8-11eb-97da-e371fd889991.png)

The javascript expression `follow.organization.id` evaluates the `id` attribute of the object stored in the `organization` attribute of the object called `follow`. You're right that this would be different for every iteration in the loop. The javascript expression `"follow.organization.id"` however is a constant string literal, and will not change.

Reply:
Oh! I see what you mean now 🤦‍♂️ Also seems like warnings was turned of in my console. Sorry about this!